src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Model/Column.cs

```diff
@@ -434,6 +434,36 @@ public override void Set(SqlDataRecord record, int ordinal, byte? value)
         }
     }
 
+    /// <summary>
+    /// Represents a rowversion type column.
+    /// </summary>
+    /// <remarks>
+    /// The timestamp data type is a synonym for the rowversion data type, and its syntax is now deprecated in SQL.
+    /// </remarks>
+    public class TimestampColumn : Column<byte[]>
+    {
+        public TimestampColumn(string name)
+            : base(name, SqlDbType.Timestamp, true)
+        {
+        }
+
+        public override byte[] Read(SqlDataReader reader, int ordinal)
+        {
+            // The rowversion storage size is 8 bytes.
+            const int length = 8;
+
+            byte[] bytes = new byte[length];
+            reader.GetBytes(Metadata.Name, ordinal, 0, bytes, 0, length);
```

Comment:
can we add named parameter to these 0s? Makes reading the code without IDE a bit easier.

Reply:
Good idea, thanks. Added in be0c10f and af5964a.